release/.buildkite/build_pipeline.py

```diff
@@ -0,0 +1,307 @@
+import copy
```

Comment:
Is it possible to put this under `ci/` instead of making a new toplevel dir?

Reply:
release is already a top level dir with all the release testing config?